extension/model/model.go

```diff
@@ -0,0 +1,17 @@
+package model
+
+import (
+	"os"
+	"path"
+)
+
+// ExtensionDir is directory path where to store the extensions
+var ExtensionDir string
+
+func init() { //nolint:gochecknoinits
+	userHomeDir, err := os.UserHomeDir()
```

Comment:
We are reading home directory in config module, is there a way we can reuse that here ?

Reply:
the only implementation about getting user home dir is this line and github parser. the other is on plugin. I cannot find in config module. could you help to point it out?

Reply:
Sorry my bad, the code to read the home directory was removed from config.